main.nf

```diff
@@ -569,7 +626,6 @@ process multiqc {
 
     output:
     file '*multiqc_report.html'
-    file '*multiqc_data'
```

Comment:
I'm thinking that we don't really need to deliver the parsed data files to the user. They can find all info from the html file. Many raw files may cause our trouble to explain everything...